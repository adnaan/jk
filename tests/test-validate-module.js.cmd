jk validate -m ./validate-files/module ./validate-files/*.yaml
