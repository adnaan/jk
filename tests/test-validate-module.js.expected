./validate-files/invalid.yaml: object name is not "Valid"
./validate-files/valid.yaml: ok
